src/test/java/teammates/test/cases/datatransfer/StudentProfileAttributesTest.java

```diff
@@ -161,9 +201,16 @@ private StudentProfile createStudentProfileFrom(
 
     private StudentProfileAttributes createNewProfileAttributesFrom(
             StudentProfileAttributes profile) {
-        return new StudentProfileAttributes(profile.googleId, profile.shortName, profile.email,
-                                            profile.institute, profile.nationality, profile.gender,
-                                            profile.moreInfo, profile.pictureKey);
+        return StudentProfileAttributes.builder()
```

Comment:
I think this can be a method in the attributes or builder class itself, e.g. `getCopy()`